### DISCUSSION TITLE
Support for the VMware produced VMWare 7.0 STIG

### DISCUSSION THREAD

**Discussion by TheTroubleshooter** [NUWCDIVNPT/stig-manager#767] [Ideas and Enhancements]
Could you please look at supporting the import of the VMware produced VMWare 7.0 STIG since DISA has not released their own yet? The VMware produced STIG imports into STIG Viewer just fine but fails on STIG Manager. 

Information can be found here: https://github.com/vmware/dod-compliance-and-automation
Documents can be found here: https://github.com/vmware/dod-compliance-and-automation/tree/master/vsphere/7.0/docs

**Comment by cd-rite** [comment]
Hi @TheTroubleshooter From just a quick look, it looks like the VMWare-produced STIGs do not have a `<fixtext>` element in ALL of their rules, which is something we have expected from experience with the DISA-produced STIGs. However, it appears the official XCCDF specification does support [0-n] fixtext elements, so we will look into the effort required to support this. 
Thank you for bringing it to our attention!

**Comment by cd-rite** [comment]
Hi @TheTroubleshooter Closing some old discussions.... We added support for those STIGs formatted in that VMWare style with Release 1.3.1
PR: #931
